app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -28,6 +38,17 @@ def create_in_provider_queue(manager_id, params)
 
     private
 
+    def notify(op, manager_id, params, success)
```

Comment:
I have https://github.com/ManageIQ/manageiq/pull/14453 in the making. 

Do you have other suggestion as to where to move this?